docs/Using-Operations-Service.md

```diff
@@ -7,6 +7,8 @@
 - [Start Periodic Polling](#start-periodic-polling)
 - [Approve an Operation](#approve-an-operation)
 - [Reject an Operation](#reject-an-operation)
+- [Detail of an Operation](#detail-of-an-operation)
+- [Claim an Operation](#claim-an-operation)
```

Comment:
Again, it should be `the` because we're claiming specific operation via ID, not a random unknown operation.